apps/mobile-alphabet/src/app/Components/alphabet-card-detail.screen.tsx

```diff
@@ -1,11 +1,14 @@
 import React, { useEffect, useState } from 'react';
 import { Button, Image, Text, View } from 'react-native';
+import Sound from 'react-native-sound';
 import config from './Config.json';
 import { AlphabetData } from './Menu';
 
 export function AlphabetCardDetailScreen() {
     const [alphabetData, setAlphabetData] = useState<AlphabetData | null>(null);
 
+    const [audioLoaded, setAudioLoaded] = useState(false);
```

Comment:
If this is a boolean flag, name it
`[hasAudioLoaded, setHasAudioLoaded]`

Reply:
Alternatively, you might want to have an audio state that includes 'idle', 'pending', 'loaded', and 'error'.